lib/yam/agama/patch_agama_base.pm

```diff
@@ -0,0 +1,24 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: base class for Patch Agama tests
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package yam::agama::patch_agama_base;
```

Comment:
this still looks strange, we should be able to use some base cases for agama, but the current one we use for autoyast includes few things we don't need.
Could you please create a new ticket to sort this out?

Reply:
https://progress.opensuse.org/issues/130573